Assets/Scripts/ItemSlot.cs

```diff
@@ -0,0 +1,64 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ItemSlot : MonoBehaviour
+{
+    private GameObject equippedItem;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        equippedItem = null;
+    }
+
+    private void Update()
+    {
+        if (IsEquipped())
+        {
+            equippedItem.transform.position = transform.position;
+        }
+    }
+    // Check if any item is equipped
+    public bool IsEquipped()
+    {
+        return !equippedItem;
+    }
+
+    // Remove equipped item and return it (return null if none is equipped)
+    public GameObject RemoveEquippedItem()
+    {
+        GameObject item = equippedItem;
+        equippedItem = null;
+        return item;
```

Comment:
Мы точно не хотим тут включать у item SpriteRenderer обратно?

Reply:
Вообще можно просто в 32-й строке позвать `UnequipItem()`.

Reply:
unequip item works via this method. I didn't put activation of sprite renderer here, as it may be required to get item without activation, for example, for deleting it after using a key to open a specific door, etc.